FelixAlexK/PawSwipe
# Feature fkuhbier like UI swp23 mtv 53

To test features, use the following account: 
email: test.kuhbier@email.com
password: password

New features:

- Provides functions that allow you to perform actions for an animal you like or dislike. Not yet linked to match activities.
- Listing of all “liked” animals, in AnimalListActivity (use the account above to see it in action).
- Images can now be uploaded to the database and also displayed (see AnimalListActivity, if you are logged in with the account above).
